packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -940,6 +940,16 @@ export default class RestApi {
         },
       },
     }
+    collections.groups.actions = {
+      create: withParams(
+        async ({ name }, req) => {
+          return (await app.createGroup({ name })).id
+        },
+        {
+          name: { type: 'string' },
+        }
+      ),
+    }
```

Comment:
this will expose `POST /rest/v0/groups/<id>/create` and that not what we want. You can remove this.